package/network/config/ipip/files/ipip.sh

```diff
@@ -32,7 +32,9 @@ proto_ipip_setup() {
 		break
 	done
 
-	( proto_add_host_dependency "$cfg" "$peeraddr" "$tunlink" )
+	if [ "${nohostroute}" != "1" ]; then
```

Comment:
I'm wondering if `!= "1"` is ideal or if you could use some _get bool_ function which also resolves `false` etc. I think I've seen it [here](https://github.com/openwrt/openwrt/blob/master/package/base-files/files/lib/functions.sh#L121).

Reply:
@aparcar would this not be pretty much the same? something like
`if [ "$(get_bool "$nohostroute" 0)" != "1" ] ; then`

If there is a better alternative I can apply it to this and the gre/wg scripts.